languages/javascript/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,20 @@
+Nullability in Javascript expresses the intentional absence of any object value. It is represented by the `null` value.
+
+```javascript
+// I do not have an apple.
+var apple = null;
+apple; // null
+
+// null is treated as falsy for boolean operations, therefore
+!apple; //true
+```
+
+`null` is part of Javascript's primitive values, and it has to be differentiated from the variable `undefined`. The
+distinction being that `undefined` is the value of an uninitialized variable or type, while `null` represents a missing object.
```

Comment:
I think this point is great, but probably belongs in the `about.md` document (formerly the `after.md` document).

```suggestion
Null values are often confused with `undefined` values.  They are two distinct primitive values in JavaScript. The distinction being that `undefined` is the value of an uninitialized variable or type, while `null` represents the absence of an object.
```